tests/NHSD.BuyingCatalogue.Ordering.Services.UnitTests/OrderItemServiceTests.cs

```diff
@@ -111,14 +111,14 @@ public static void Constructor_NullAccessor_ThrowsArgumentNullException()
 
         [Test]
         [InMemoryDbAutoData]
-        public static async Task GetOrderItems_ReturnsEmpty(
+        public static async Task GetOrderItems_ReturnsNull(
             CallOffId callOffId,
             CatalogueItemType catalogueItemType,
             OrderItemService service)
         {
```

Comment:
A pre condition/check to prove that the `callOffId` is invalid is recommended to prove that there is no item with the Id in the database.
i.e. `(await context.Order.AnyAsync(o => o.Id == callOffId.Id)).Should().BeFalse();`

Reply:
With PR [424](https://github.com/nhs-digital-gp-it-futures/BuyingCatalogueOrdering/pull/424/files)
Checked existence of Order with invalid CallOfId